showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITDynamicRoutingHeaders.java

```diff
@@ -175,6 +189,24 @@ public void testGrpc_matchesHeaderName() {
     assertThat(requestHeaders).containsExactlyElementsIn(expectedHeaders);
   }
 
+  @Test
+  public void testGrpc_implicitHeadersEnum() {
```

Comment:
nit: Rename this to be a bit more descriptive. Something like `testGrpc_implicitHeaders_enumsEncodedasInt()` to describe that this test covers specifically enums -> int.

Reply:
Also, can you add an httpjson test case as well. I think it affects both transports. Ignore this comment if it only affects gRPC.

Reply:
> nit: Rename this to be a bit more descriptive. Something like `testGrpc_implicitHeaders_enumsEncodedasInt()` to describe that this test covers specifically enums -> int.

+1000 to this. See [unit tests naming](https://engdoc.corp.google.com/eng/doc/devguide/testing/unit/best-practices.md?cl=head&polyglot=java#naming) for details.

Reply:
renamed the tests and added httpjson case as well. thank you.